tests/ui/attr-value-is-invalid.rs

```diff
@@ -0,0 +1,23 @@
+use bilge::prelude::*;
+
+// negative value
+#[bitsize(-1)]
+enum Test {}
+
```

Comment:
might want to check `0` as well, since we used to allow it

Reply:
it's commented out because I started when the change wasn't merged 👍